src/app/pages/observation-list/observation-list.page.html

```diff
@@ -65,17 +65,19 @@ <h3 class="ion-text-uppercase header">{{'OBSERVATION_LIST.TITLE' | translate }}<
 
   <ng-container *ngIf="!(isFetchingObservations$ | async); else skeleton">
     <ng-container *ngIf="viewType$ | async as viewType">
-      <ng-container *ngIf="registrations$ | async as registrations">
-        <ion-list class="loaded" *ngIf="registrations?.length > 0; else empty">
-          <app-observation-list-view
-            *ngIf="viewType==='list'"
-            [registrations]="registrations"
-          ></app-observation-list-view>
-          <app-images-grid *ngIf="viewType==='grid'" [attachments]="registrations"></app-images-grid>
-          <ion-infinite-scroll (ionInfinite)="loadNextPage()" [disabled]="shouldDisableScroller$ | async">
-            <ion-infinite-scroll-content> </ion-infinite-scroll-content>
-          </ion-infinite-scroll>
-        </ion-list>
+      <ng-container *ngIf="showObservations$ | async">
```

Comment:
Jeg tror det blir litt rart å ikke vise observasjoner i listevisninga? Da blir det jo bare en helt tom side. 
Jeg tror vi bør lage en empty state som forklarer at man må skru togglen på for å se noe i lista. Se hvordan empty-staten under er laget.

Reply:
Dette fiksa du jo fint Jørgen med din commit. Synes det ble ryddig.